.github/workflows/deploy.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           channel-id: ${{ secrets.SLACK_RELEASE_CHANNEL_ID }}
           payload: |
             {
-              "text": "*<${{env.RELEASES_URL}}v${{ inputs.version }}|v${{ inputs.version }}>*\nCC: <@U03KG4BK1L1> <@U02AE5GMMHV> <@U01LVJ30QEB> <@U03NAD30MTJ> <@U02AGPP420Y> <@U035SSC9NSH>",
+              "text": "*<${{env.RELEASES_URL}}v${{ inputs.version }}|v${{ inputs.version }}>*\nCC: <@U03KG4BK1L1> <@U02AE5GMMHV> <@U01LVJ30QEB> <@U024YF8CR53> <@U02AGPP420Y> <@U035SSC9NSH>",
```

Comment:
@krishna2020, we can also refer to Slack user groups here. It'd be easier to make changes in Slack than have to raise PRs each time.
I'd recommend creating separate owner groups for each team and referring them here.

Feel free to skip and resolve this comment, if you don't agree.

Reply:
I was not aware of groupId, that would be far better

Reply:
The syntax is incorrect. It should be of the format, `<!subteam^GROUP_ID>`.